float32 robot1
float32 robot2
float32 robot3
int32 laptop


